e2e/cypress/integration/system_console/compliance/custom_terms_of_service_spec.js

```diff
@@ -50,4 +61,85 @@ describe('Custom Terms of Service', () => {
+
+        // # Login as admin
+        cy.apiAdminLogin();
+
+        // # Reset config via the API from previous test
+        cy.apiUpdateConfig({
+            EmailSettings: {
+                RequireEmailVerification: false,
+            },
+            SupportSettings: {
+                CustomTermsOfServiceEnabled: false,
+            },
+        });
+
+        // # Visit custom terms of service page
+        cy.visit('/admin_console/compliance/custom_terms_of_service');
+
+        // # Enable custom terms of service
+        cy.findByTestId('SupportSettings.CustomTermsOfServiceEnabledtrue').click();
+
+        // # Set the terms of service to the first value
+        cy.findByTestId('SupportSettings.CustomTermsOfServiceTextinput').clear().type(firstTOS);
+
+        // # Save config
+        cy.get('#saveSetting').click();
+
+        // # Accept the terms as sysadmin
+        cy.get('#acceptTerms').should('be.visible').click();
+
+        // * Should be redirected back to custom terms of service page
+        cy.url().should('include', `/admin_console/compliance/custom_terms_of_service`);
+
+        // # Login as the test user
+        cy.apiLogin(testUser);
+
+        // # Visit the test team town square
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // * Ensure that the first terms of service is visible
+        cy.findByTestId('termsOfService').should('be.visible').and('contain.text', firstTOS);
+
+        // # Login as admin
+        cy.apiAdminLogin();
+
+        // # Visit custom terms of service page
+        cy.visit('/admin_console/compliance/custom_terms_of_service');
+
+        // # Set the terms of service to the first value
+        cy.findByTestId('SupportSettings.CustomTermsOfServiceTextinput').clear().type(secondTOS);
+
+        // # Save config
+        cy.get('#saveSetting').click();
+
+        // * Ensure that the terms of service text shows as expected
+        cy.findByTestId('termsOfService').should('be.visible').and('contain.text', secondTOS);
+
+        // # Accept the terms as sysadmin
+        cy.get('#acceptTerms').should('be.visible').click();
+
+        // * Should be redirected back to custom terms of service page
+        cy.url().should('include', `/admin_console/compliance/custom_terms_of_service`);
+
+        // # Login as the test user
+        cy.apiLogin(testUser);
+
+        // # Visit the test team town square
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // * Ensure that the new terms of service is visible
+        cy.findByTestId('termsOfService').should('be.visible').and('contain.text', secondTOS);
+
+        // * Ensure that the accept terms button is visible and click it
+        cy.get('#acceptTerms').should('be.visible').click();
+
+        // * Ensure the user is redirected to the appropriate team after terms are accepted
+        cy.url().should('include', `/${testTeam.name}/channels/town-square`);
+    });
```

Comment:
This test case has these steps at the end... 

- At this point, both users are seeing the TOS page (system admin is still in System Console)
- Ordinary user, UserA, click on "I Agree" on Terms of Service page you are viewing
- UserA then sees the updated TOS from step 4
- UserA, click on "I Agree" again

however I dont think this can be done with cypress, what i have done instead is just ensured that the first terms of service is visible, logout as regular user and then after creating second terms of service as admin login again as regular user and accept the updated terms of service